### PR TITLE
Feature/devas 666 text around video

### PR DESCRIPTION
**Pull request by ayyazdaniaryan**
Add a bottom level description to a video.

### CODE REVIEW

File: assembl/tests/fixtures/graphql.py [hunk 1]
```diff
@@ -61,10 +61,14 @@ def thematic_with_video_and_question(graphql_request):
                 {value:"Laurent Alexandre, chirurgien et expert en intelligence artificielle nous livre ses prédictions pour le 21e siècle.",
                  localeCode:"fr"},
             ]
-            descriptionEntries:[
+            descriptionEntriesTop:[
                 {value:"Personne ne veut d'un monde où on pourrait manipuler nos cerveaux et où les états pourraient les bidouiller",
                  localeCode:"fr"},
             ],
+            descriptionEntriesBottom:[
+                {value:"Calise de tabarnak",
```
Comment (swergas) [inline assembl/tests/fixtures/graphql.py]: :D

Comment (maparent) [inline assembl/tests/fixtures/graphql.py]: It's "câlisse" :-)

File: assembl/static2/js/app/actions/adminActions.js [hunk 2]
```diff
@@ -77,8 +77,12 @@ export const updateVideoHtmlCode = (id, value) => {
...
-  return { id: id, locale: locale, value: value, type: 'UPDATE_VIDEO_DESCRIPTION' };
+export const updateVideoDescriptionTop = (id, locale, value) => {
+  return { id: id, locale: locale, value: value, type: 'UPDATE_VIDEO_DESCRIPTION_TOP' };
+};
+
+export const updateVideoDescriptionBottom = (id, locale, value) => {
+  return { id: id, locale: locale, value: value, type: 'UPDATE_VIDEO_DESCRIPTION_BOTTOM' };
 };
 
 export const updateVideoTitle = (id, locale, value) => {
```
Comment (paolina92) [inline assembl/static2/js/app/actions/adminActions.js]: Add a test please

File: assembl/static2/js/app/reducers/adminReducer.js [hunk 3]
```diff
@@ -137,7 +139,8 @@ export const thematicsHaveChanged = (state = false, action) => {
   case 'UPDATE_THEMATIC_IMG_URL':
   case 'UPDATE_THEMATIC_TITLE':
   case 'TOGGLE_VIDEO':
-  case 'UPDATE_VIDEO_DESCRIPTION':
+  case 'UPDATE_VIDEO_DESCRIPTION_TOP':
+  case 'UPDATE_VIDEO_DESCRIPTION_BOTTOM':
   case 'UPDATE_VIDEO_HTML_CODE':
   case 'UPDATE_VIDEO_TITLE':
     return true;
```
Comment (paolina92) [inline assembl/static2/js/app/reducers/adminReducer.js]: Same here. Add a test ;)

File: assembl/static2/js/app/components/administration/survey/videoForm.jsx [hunk 4]
```diff
@@ -103,8 +123,11 @@ export const mapDispatchToProps = (dispatch, { selectedLocale, thematicId }) =>
...
-    updateDescription: (value) => {
-      return dispatch(updateVideoDescription(thematicId, selectedLocale, value));
+    updateDescriptionTop: (value) => {
+      return dispatch(updateVideoDescriptionTop(thematicId, selectedLocale, value));
+    },
+    updateDescriptionBottom: (value) => {
+      return dispatch(updateVideoDescriptionBottom(thematicId, selectedLocale, value));
     },
     updateTitle: (value) => {
       return dispatch(updateVideoTitle(thematicId, selectedLocale, value));
```
Comment (paolina92) [inline assembl/static2/js/app/components/administration/survey/videoForm.jsx]: This component should be easy to test. We can try?

File: assembl/tests/fixtures/graphql.py (hunk 1 above)
Comment (ayyazdaniaryan) [inline assembl/tests/fixtures/graphql.py]: Bahaha fixed!

File: assembl/static2/js/app/reducers/adminReducer.js (hunk 3 above)
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/reducers/adminReducer.js]: Done

**Pull request merged**